package.json

```diff
@@ -7,26 +7,28 @@
     "dependencies": {
         "@grpc/grpc-js": "^1.2.7",
         "@grpc/proto-loader": "^0.6.4",
+        "fs-extra": "^10.0.1",
         "long": "^4.0.0",
         "minimist": "^1.2.5",
+        "mock-fs": "^5.1.2",
```

Comment:
This should be a dev dependency, not a dependency

Reply:
Good call lol thanks!